scripts/generate/clean-up.ts

```diff
@@ -8,6 +8,7 @@ export async function cleanUp({template}: {template: Template}) {
   exec(`rm -rf ${path.resolve('.', 'scripts')}`);
   exec(`rm -rf ${path.resolve('.', '.prettierrc')}`);
   exec(`rm -rf ${path.resolve('.', '.eslintrc.js')}`);
+  exec(`rm -rf ${path.resolve('.', '.github')}`);
```

Comment:
Should we remove `.gitignore`?

Reply:
The `.gitignore` has `node_modules/` and `build/`. I think it wouldn't be terrible to keep!